src/pages/Skills/Skills.js

```diff
@@ -6,10 +6,9 @@ import {
   EuiPageHeaderSection,
   EuiTitle,
   EuiBasicTable,
-  EuiIcon,
   EuiText
 } from "@elastic/eui";
-
+import sortTableValues from "../../utils/sortTableValues";
```

Comment:
Based on the comments below this should be `import { sortTableValues }....`

Reply:
Updated